repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/LauncherUtils.java

```diff
@@ -332,7 +326,6 @@ public static FlaggedOption defineArgDockerImageName() {
         opt.setShortFlag('n');
         opt.setLongFlag("name");
         opt.setStringParser(JSAP.STRING_PARSER);
-        opt.setRequired(true);
```

Comment:
Since the `required` flag is removed we should probably set a default of `spirals/repairnator:latest`.